lib/ruby/truffle/mri/bigdecimal.rb

```diff
@@ -0,0 +1,9 @@
+# TODO require bigdecimal/*
+
+BigDecimal = Truffle::BigDecimal
+
+module Kernel
+  def BigDecimal(*args)
+    BigDecimal.new *args
+  end
+end
```

Comment:
I think this should live in `lib/ruby/truffle/truffle/bigdecimal.rb`. `mri` is for files that just point to MRI stdlib code.